salt/modules/dig.py

```diff
@@ -99,6 +99,36 @@ def A(host, nameserver=None):
     return [x for x in cmd["stdout"].split("\n") if check_ip(x)]
 
 
+def X(host, nameserver=None):
```

Comment:
Are you using the name `X` because that is the arg passed to dig?

I think it would make more sense to name the function PTR, since that is the type of record being returned.